ccmlib/scylla_docker_cluster.py

```diff
@@ -99,12 +110,13 @@ def update_yaml(self):
         with open(conf_file, 'w') as f:
             yaml.safe_dump(data, f, default_flow_style=False)
 
-    def create_docker(self):
+    def create_docker(self, args=None):
```

Comment:
so, to avoid what @Orenef11 mentions, we could start it as dict() or even an empty string

Reply:
@fgelcer @ShlomiBalalis 
Please dont method with `dict` or `list` variable because it's wrong in Python:
```
def blablab(a=[]):
   ...:     a.append(1)
   ...:     print(a)
   ...: 
blablab()
[1]
blablab()
[1, 1]
blablab()
[1, 1, 1]
blablab()
[1, 1, 1, 1]
```

It's the correct way:
```
     def create_docker(self, args=None):
        args = args or {}
```